docs/docs/cmd/tenant/serviceannouncement/serviceannouncement-health-list.md

```diff
@@ -119,3 +119,19 @@ When we make use of the option `issues` the response will differ.
     id,status,service
     Exchange,serviceDegradation,Exchange Online
     ```
+
+=== "Markdown"
```

Comment:
Also for this command, we have to update 2 responses (there's another one a bit above)